### DISCUSSION TITLE
Someone did a feature release while I was working on alpha

### DISCUSSION THREAD

**Discussion by anshumanpriyadarshi** [semantic-release/semantic-release#3378] [Release workflow] [answered]
I was working on a alpha branch (not a BREAKING CHANGE) to upgrade. While I had my alpha release `v1.7.0-alpha.1`  to be merged to `main` for a `v1.7.0`, someone did a release in main and its already in `1.7.0`.

What should be the way forward from here? I am unable to merge `main` to `alpha` because of the conflicts in CHANGELOG.

NOTE: `alpha` is a protected branch.

Please help

**Comment by travi** [comment, marked as answer]
Merge main to alpha and resolve conflicts manually like any other merge with conflicts

**Reply by travi** [reply]
Also consider https://semantic-release.gitbook.io/semantic-release/support/faq#should-release-notes-be-committed-to-a-changelog.md-in-my-repository-during-a-release in order to avoid such conflicts in the future
